qml/TimeZonePage.qml

```diff
@@ -518,6 +547,12 @@ BasePage {
         }
     }
 
+    Timer {
+        interval: 1000
```

Comment:
For me it refreshes only visible delegates (about 10 per sec), which is ok, i think.
It would also look not good if time in list and system time are different for a long time.